opencog/nlp/relex2logic/rule-helpers.scm

```diff
@@ -631,7 +631,7 @@
 	(cond
 		((or (string=? determiner "those") (string=? determiner "these"))
 			(ListLink
-				(ImplicationLink
+				(ImplicationScopeLink
```

Comment:
this looks correct -- it looks like a classic example of where scoping is needed and is important.